packages/reacord/library/core/components/action-row.tsx

```diff
@@ -44,4 +45,12 @@ class ActionRowNode extends Node<{}> {
       child.modifyMessageOptions(options)
     }
   }
+  override handleComponentInteraction(interaction: ComponentInteraction) {
```

Comment:
There's a bug with action rows where if you use them in your code, they won't pass the interaction down to their children resulting in this event being missed - This would be good to have merged in